src/rgw/librgw.cc

```diff
@@ -489,6 +491,8 @@ namespace rgw {
     rgw_tools_init(g_ceph_context);
 
     rgw_init_resolver();
+    rgw::curl::setup_curl(boost::none);
+    rgw_http_client_init(g_ceph_context);
```

Comment:
it looks like the rgw_http_client_init/cleanup() methods came from https://github.com/ceph/ceph/pull/18932 which wasn't backported to luminous, so those two calls can be removed